contracts/CertCoin.sol

```diff
@@ -26,9 +32,9 @@ contract CertCoin {
 
         balanceOf[msg.sender] = 10000000000;                   // Give the creator all initial tokens
         totalSupply = 10000000000;                             // Update total supply
-        name = 'CertCoin';                                   // Set the name for display purposes
```

Comment:
I personally like single quotes.

Reply:
@olegnatsevsky i've been making all these changes according vs code's solidity extension linter
and actually style guide provided below recommends using double-quotes :)